src/DatabaseInfo.py

```diff
@@ -0,0 +1,65 @@
+import pandas as pd
+from ast import literal_eval
+from urllib.request import urlopen
+import json
+from io import StringIO
+
+
+
+# BiGG

```

Comment:
Why do we need to read the BiGG db twice?

Reply:
The idea is to read BiGG and save the needed info from its API in a file only once, because it's time consuming. Then, we can read BiGG and the saved file when needed for models written in BiGG namespace.